From aa101165c2144d30cf8b76c55a218cfa65486ee9 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 8 Jul 2016 15:30:33 +0800
Subject: [PATCH 1109/5242] MLK-12975-2 video: mxsfb: missing a 'break' in the
 case statement.

commit  154862c0221bafa185b7ed7abf2da248eab35b57 from
https://source.codeaurora.org/external/imx/linux-imx.git

Missing the 'break' will cause the function return failure
when the 'bits_per_pixel' is 32 which should be supported.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index f8bfcc4..2da4196 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1040,6 +1040,7 @@ static int mxsfb_restore_mode(struct fb_info *fb_info)
 		break;
 	case 3:
 		bits_per_pixel = 32;
+		break;
 	case 1:
 	default:
 		return -EINVAL;
-- 
1.7.9.5

